client/updateCenter.go

```diff
@@ -68,20 +71,23 @@ type InstallStatesJob struct {
 	Version         string
 }
 
-func (u *UpdateCenterManager) Status() (status *UpdateCenter, err error) {
-	api := fmt.Sprintf("%s/updateCenter/api/json?pretty=false&depth=1", u.URL)
-	var (
-		req      *http.Request
-		response *http.Response
-	)
-
-	req, err = http.NewRequest("GET", api, nil)
-	if err == nil {
-		u.AuthHandle(req)
-	} else {
-		return
-	}
+type CenterPlugin struct {
+	CompatibleWithInstalledVersion bool
+	excerpt                        string
+	Installed                      InstalledPlugin
+	minimumJavaVersion             string
+	Name                           string
+	RequiredCore                   string
+	SourceId                       string
+	Title                          string
+	URL                            string
+	Version                        string
+	Wiki                           string
+}
 
+func (u *UpdateCenterManager) Status() (status *UpdateCenter, err error) {
```

Comment:
A comment is needed. See also https://goreportcard.com/report/github.com/jenkins-zh/jenkins-cli#golint